client/src/components/Home/HomeGridCurrentGlmLock/ModalLockGlm/LockGlmTabsInputs/LockGlmTabsInputs.tsx

```diff
@@ -0,0 +1,149 @@
+import cx from 'classnames';
+import React, { forwardRef, useDeferredValue, useEffect, useState } from 'react';
+
+import InputText from 'components/ui/InputText';
+import useCryptoValues from 'hooks/queries/useCryptoValues';
+import useSettingsStore from 'store/settings/store';
+import {
+  comma,
+  floatNumberWithUpTo18DecimalPlaces,
+  floatNumberWithUpTo2DecimalPlaces,
+} from 'utils/regExp';
+
+import styles from './LockGlmTabsInputs.module.scss';
+import LockGlmTabsInputsProps from './types';
+
+const LockGlmTabsInputs = forwardRef<HTMLInputElement, LockGlmTabsInputsProps>(
```

Comment:
This component is a copy of `EarnGlmLockTabsInputs`. Do we keep the source, after it's already moved?

Reply:
I have a plan to remove every unused part of the app after finishing each view. Are you ok with it?

Reply:
I would remove it already if it's not used, but we can keep it too.